### PR TITLE
Reword the landing page as per OMC vote.

### PR DESCRIPTION
**Pull request by paulidale**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -12,16 +12,21 @@
...
-          <a href="/community">community</a> page. To get the
-          latest news, download the source, and so on, please see
-          the sidebar or the buttons at the top of every page.
+	  The OpenSSL Project develops and maintains the OpenSSL
+	  software - a robust, commercial-grade, full-featured toolkit
+	  for general-purpose cryptography and secure communication. The
+	  project's technical decision making is managed by the OpenSSL
+	  Technical Committee (OTC) and the project governance is managed
+	  by the OpenSSL Management Committee (OMC). The project operates
+	  under formal Bylaws.
```
Comment (mattcaswell) [inline index.html]: I'd suggest making "OpenSSL Technical Committe", "OpenSSL Management Committee" and "Bylaws" links to the appropriate page.

**Review by mattcaswell — CHANGES_REQUESTED** [review]

File: index.html [hunk 2]
```diff
@@ -12,16 +12,22 @@
...
-          latest news, download the source, and so on, please see
-          the sidebar or the buttons at the top of every page.
+	  The OpenSSL Project develops and maintains the OpenSSL
+	  software - a robust, commercial-grade, full-featured toolkit
+	  for general-purpose cryptography and secure communication. The
+	  project's technical decision making is managed by the
+          <a href="/community/otc.html">OpenSSL Technical Committee</a> (OTC)
+          and the project governance is managed by the
+          <a href="/community/omc.html">OpenSSL Management Committee</a> (OMC).
+          The project operates under formal Bylaws.
```
Comment (t-j-h) [inline index.html]: Bylaws should be a link to /policies/omc-bylaws.html

Comment (levitte) [reply index.html]: Side note: the file omc-bylaws.html should be renamed to bylaws.html...

**Review by t-j-h — APPROVED** [review]

**Review by mattcaswell — APPROVED** [review]

**Review by levitte — APPROVED** [review]

**Comment by paulidale** [comment]
Merged, thanks for the feedback.

**Pull request closed without merging**